### DISCUSSION TITLE
Distribution via conda

### DISCUSSION THREAD

**Discussion by tgkolda** [sandialabs/pyttb#335] [General]
Hi everyone,

I know pyttb is available via pip, but is there a way to make it available via the conda install? I'm able to install everything else that way. 

Thanks for any discussion.

-Tammy

**Comment by dmdunla** [comment]
Using pip for installs should work everywhere Python is available, whereas conda will only work in conda environments. We decided to support pip to start but would be open to support conda packaging if it makes sense.

@tgkolda Can you add an example of your install workflow here? Is pip not working? Typically for pip-only install packages, the typical workflow for Cinda users is to create a conda env, activate the ebb, then install the PyPI packages that are needed. If support is needed for something else, can you add some notes here?

**Reply by tgkolda** [reply]
I think this is my ignorance about conda install and pip. Googling around a bit, it seems the consensus is that you get your conda environment set up (if you're using one) and then can use pip within that environment. My confusion was that I was able to install every other package using conda, so I had some impression that all packages should be available via that route.
